featurebyte/query_graph/node/metadata/sdk_code.py

```diff
@@ -478,6 +509,43 @@ def generate(self, to_format: bool = False) -> str:
             elif isinstance(statement, str):
                 statement_lines.append(statement)
         statements = "\n".join(statement_lines)
+        return statements, import_pairs
+
+    def generate(
+        self,
+        to_format: bool = False,
+        output_var_name: Optional[str] = None,
+        skip_remove_unused_variables: bool = False,
```

Comment:
nit: what do you think about having the option as `remove_unused_variables = True`? Seems slightly easier to understand without the double negatives.

Reply:
Will update this to avoid double negatives.